lucien-ios/Add New Comic Book/AddComicViewController.swift

```diff
@@ -0,0 +1,224 @@
+//
+//  AddComicViewController.swift
+//  lucien-ios
+//
+//  Created by Ahmed, Guled on 9/26/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import UIKit
+import QuartzCore
+
+final class AddComicViewController: UIViewController, UIPickerViewDelegate, UIPickerViewDataSource, UITextFieldDelegate {
+    // MARK: - Private IBOutlets
+
+    @IBOutlet private weak var coverPhotoLabel: UILabel!
+    @IBOutlet private weak var coverPhotoButton: UIButton!
+    @IBOutlet private weak var comicTitleLabel: UILabel!
+    @IBOutlet private weak var comicTitleTextField: UITextField!
+    @IBOutlet private weak var storyTitleLabel: UILabel!
+    @IBOutlet private weak var storyTitleTextField: UITextField!
+    @IBOutlet private weak var volumeLabel: UILabel!
+    @IBOutlet private weak var volumeTextField: UITextField!
+    @IBOutlet private weak var issueLabel: UILabel!
+    @IBOutlet private weak var issueTextField: UITextField!
+    @IBOutlet private weak var publisherLabel: UILabel!
+    @IBOutlet private weak var publisherTextField: UITextField!
+    @IBOutlet private weak var releaseDateLabel: UILabel!
+    @IBOutlet private weak var genreLabel: UILabel!
+    @IBOutlet private weak var conditionLabel: UILabel!
+    @IBOutlet private weak var selectADateButton: UIButton!
+    @IBOutlet private weak var selectAGenreButton: UIButton!
+    @IBOutlet private weak var selectAConditionButton: UIButton!
+    @IBOutlet private weak var datePicker: UIDatePicker!
+    @IBOutlet private weak var conditionPicker: UIPickerView!
+    @IBOutlet private weak var genrePicker: UIPickerView!
+    @IBOutlet private weak var scrollView: UIScrollView!
+    @IBOutlet private weak var releaseDateBottomLine: UIView!
+    @IBOutlet private weak var genreBottomLine: UIView!
+    @IBOutlet private weak var conditionBottomLine: UIView!
+    @IBOutlet private weak var comicTitleWarningLabel: UILabel!
+
+    // MARK: - Private Variables
+    private var didCompleteForm = false
+    private var finishButton = UIBarButtonItem()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureNavigationController()
+        configureViewController()
+    }
+
+    // MARK: - Private Instance Methods
+
+    @objc private func backButtonTapped() {
+        dismiss(animated: true, completion: nil)
+    }
+
+    private func configureNavigationController() {
+        guard let navigationController = navigationController else {
+            return
+        }
+
+        // NavigationBar Background
+
+        navigationController.navigationBar.isTranslucent = true
```

Comment:
Every section that's more than one line and headed by a comment should be extracted into a function.